### DISCUSSION TITLE
Clustering customization in reagraph

### DISCUSSION THREAD

**Discussion by natalya-k-dblue** [reaviz/reagraph#141] [Q&A] [answered]
Hi everyone, 
I am looking for the possibility to visually group certain nodes.  I see that there is a clusterAttribute property on GraphCanvas that visualizes clusters with a circle around clustered nodes which has a radius depending on the nodes number and size: https://reagraph.dev/?path=/docs/docs-advanced-clustering--docs.  I am looking at the possibility to customize clustering representation: e.g. with a rectangle instead of a circle (because i am using custom nodes that are rectangles) or setting the circle radius in the code. Is there a way to do so in reagraph? Or maybe there are alternative ways to go around visually grouping nodes?

**Comment by amcdnl** [comment, marked as answer]
I'm open to this - it would need to be built in though given the complexity of the calculations. We had a issue/PR opened with a strawman implementation but it was not completed.

**Reply by natalya-k-dblue** [reply]
Thank you for the response. I will check it out.
